src/Input/Input.js

```diff
@@ -67,6 +67,12 @@ export const styleSheet = createStyleSheet('MuiInput', (theme) => {
         appearance: 'none',
       },
     },
+    multiLine: {
+      resize: 'none',
+      'line-height': 'inherit',
+      padding: '0px',
+      'margin-top': '12px',
```

Comment:
I think all 12px of margin needs to be on top of the element otherwise the text will overlap with the label for the field when there's overflow. Happy to try it this way though and see if that works